app/liquid/liquid_renderer.rb

```diff
@@ -134,6 +134,7 @@ def isolate_from_plugin_data(field)
 
   def location
     return @location if @location.blank?
+    return {} if @location.data['country_code'] == 'RD'
```

Comment:
This code was written for a previous iteration of the javascript, in which the champaign.personalization.member.country country overrides the champaign.personalization.location.country, which you can see spec'd originally [here](https://github.com/SumOfUs/Champaign/commit/8ad5aac7a4c51da48c3c1c236a73f4382a9b0465#diff-b16f13723239dabec07e773b5431e0eaR115). That version is still used for petitions, but for call tool and fundraisers is now set in react and I'm not sure how we're actually handling it.